### PR TITLE
✨ Footer improvements

### PR DESCRIPTION
**Pull request by olyism**
Add more support options in the footer — i.e. Slack and contact email

- Add email support and slack link to footer.
- Optionally render #test-analytic
- Show Slack channel on TA pages

## Screenshots

Screen Shot 2022-06-06 at 4 06 29 pm

Screen Shot 2022-06-06 at 4 07 05 pm

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<footer class="footer">
+  <div>
+    <div class="footer__heading">Question or feedback</div>
+    <p>Spotted a typo? Something missing?</p>
+    <p>Please <a href="https://github.com/buildkite/docs/issues">open an issue</a> or <a href="<%= open_source_url %>">contribute an update</a>.</p>
+  </div>
+  <div>
+    <div class="footer__heading">Get help</div>
```
Comment (meiqimichelle) [inline app/views/layouts/_footer.html.erb:8]: These headings didn't read clearly to me, now that there are two.

I saw: "go to docs repo issues for feedback [on the product]", to get help go to the community Slack.

Could these be re-written so the stuff on the left is for docs-related feedback, and the path on the right for product-related? ie -- I want to encourage feedback in the Slack channel, too!

**Review by lachlanhardy — APPROVED** [review]
Code-wise, this LGTM :shipit:

**Comment by karensawrey** [comment]
@meiqimichelle and @olyism - I tried rewording the double footer to address the concerns above and the fact that the ul was tripping me up a bit. 
MY-OCD-NOOOOO

I'm absolutely sure I broke something in the process & I'm very much open to iterating this further. Please let me know what you think.

**Comment by karensawrey** [comment]
Ah, and I have a strong opinion on using "Get help" :D
no-get-help

**Review by karensawrey — CHANGES_REQUESTED** [review]

File: app/views/layouts/_footer.html.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (meiqimichelle) [reply app/views/layouts/_footer.html.erb:8]: Yes, please! 😍

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<footer class="footer">
+  <div>
+    <div class="footer__heading">Question or feedback</div>
+    <p>Spotted a typo? Something missing?</p>
```
[the remark this answers is not in the document]
Comment (meiqimichelle) [reply app/views/layouts/_footer.html.erb:4]: Idea: what about removing "questions or feedback" and use this line ("Spotted a typo? Is something missing in the docs?") as the heading?

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <div class="footer__heading">Get help</div>
+    <ul>
+      <% unless "test-analytics".in? request.path %>
+        <li>Join the <a href="https://chat.buildkite.community/">community Slack channel</a></li>
+        <li><a href="mailto:support@buildkite.com">Email support</a></li>
+      <% else %>
+        <li>Join the #test-analytics <a href="https://chat.buildkite.community/">community Slack channel</a></li>
+        <li><a href="mailto:support+analytics@buildkite.com">Email support</a></li>
+      <% end %>
+    </ul>
```
[the remark this answers is not in the document]
Comment (meiqimichelle) [reply app/views/layouts/_footer.html.erb:17]: This is v friendly and nice :) I think keeping this addition will depend on what you think of my suggestion in this comment https://github.com/buildkite/docs/pull/1572#discussion_r890626102 -- if we elevate the subhead there, then may no longer have a place for one here.

**Comment by olyism** [comment]
Thanks everyone for your suggestions, they're great improvements 🤩 I've committed them.

I've also pulled this back into draft to make some visual improvements 💄 💅 ✨

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (olyism) [reply app/views/layouts/_footer.html.erb]: Hi @meiqimichelle, for context and where my brain's at 🧠 🌈… on top of feedback, another use for the right side product-related area is support. So when customers need to get more help but Docs didn't cover enough content for it.

I like @karensawrey's new heading "Reach out", as it's inclusive of product-related feedback and getting support via email.

Lastly (out of scope for this PR), when we have the "rate how helpful this article is" feature, I'm thinking of including it in this area too.

Seems like we're happy with @karensawrey 's suggestion further down the thread, I'll mark this conversation as resolved. Feel free to raise other ideas as needed.

**Comment by olyism** [comment]
Ready for review again. Pushed up visual design improvements:

- Change background color so it stands out
- Decorate links with icons
- Hover effect

Screen Shot 2022-06-07 at 3 10 59 pm

Screen Shot 2022-06-07 at 3 11 23 pm

Screen Shot 2022-06-07 at 3 11 48 pm

Screen Shot 2022-06-07 at 3 12 22 pm

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  </section>
+  <section>
+    <div class="footer__heading">Reach out</div>
+    <p>We'd love to know what you think!</p>
+    <ul class="footer__list">
+      <li class="footer__list-item">
+        <a class="footer__list-item-link" href="https://chat.buildkite.community/">
+          <img alt="Icon: Slack" class="footer__list-item-icon" src="<%= image_url('icons/slack.svg') %>" />
+          <% unless "test-analytics".in? request.path %>
+            Join the community Slack channel
```
Comment (meiqimichelle) [inline app/views/layouts/_footer.html.erb:28]: Very minor suggestion: "Join the Buildkite community Slack" for the default

Comment (olyism) [reply app/views/layouts/_footer.html.erb]: pushed a commit to fix :)

**Review by karensawrey — APPROVED** [review]
LOVE IT!

**Pull request merged**